packages/components/src/buttons/src/ButtonGroup.module.css

```diff
@@ -0,0 +1,47 @@
+.hop-ButtonGroup {
+    display: inline-flex;
+    gap: var(--hop-space-inline-md);
+    flex-wrap: nowrap;
```

Comment:
do we really want a no wrap here? at the expense of overflowing content in some situations.

Reply:
There is a "wrap" props on ButtonGroup, which is true by default.